### PR TITLE
add tutorial

### PR DESCRIPTION
**Pull request by capital-G**
wrote a small tutorial for the beta test.

### CODE REVIEW

**Comment by capital-G** [comment]
Style of the tutorial goes like this

image


Also improved some other sections within the docs

image

File: docs/tutorial/01-story_graph.rst [hunk 1]
```diff
@@ -0,0 +1,130 @@
+.. _Story Graph:
+
+The story graph
+===============
+
+The story graph acts as a central element of Gencaster to create and control the non-linear storytelling experiences of Gencaster.
+It creates and negates possibilities through connections and statements within this graph.
```
Comment (vin-ni) [inline docs/tutorial/01-story_graph.rst:7]: negates or negotiates?

File: docs/tutorial/02-listening.rst [hunk 2]
```diff
@@ -0,0 +1,43 @@
+Listening to a graph
+====================
+
+Building a story graph is one thing, but just as important is listening to the graph.
+Gencaster provides a player for a story graph via a website, which makes it possible to distribute the content of the story graph via mobile devices to any place where there is mobile coverage.
```
Comment (vin-ni) [inline docs/tutorial/02-listening.rst:5]: Gencaster provides a player for a story graph via a website, which makes it possible to distribute the content of the story graph to any internet-enabled device.

File: docs/tutorial/02-listening.rst [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+Gencaster can also manage not only one stream, but multiple streams at the same time, so that each listener gets his or her own stream.
```
Comment (vin-ni) [inline docs/tutorial/02-listening.rst:7]: gets their own stream.

**Review by vin-ni — CHANGES_REQUESTED** [review]
This is absolutely fantastic!
Thank you!

File: docs/tutorial/01-story_graph.rst (hunk 1 above)
Comment (capital-G) [reply docs/tutorial/01-story_graph.rst:7]: negates - making everything possible may seem desirable, but actually good software omits possibilities in order to express a certain idea more easily :)

**Comment by capital-G** [comment]
@vin-ni I adjusted the things you suggested - is it OK to merge now b/c the PR is on "change requested"

**Review by vin-ni — APPROVED** [review]
Approved <3

**Pull request merged**